templates/celery.conf.j2

```diff
@@ -0,0 +1,9 @@
+; {{ ansible_managed }}
+
+[program:celery]
+autostart=true
+autorestart=true
+command = {{ crosshair_virtualenv }}/bin/celery_start
+user = celery
+stdout_logfile = {{ crosshair_celery_log }}
+redirect_stderr = true
```

Comment:
Yep, looks like it created an empty log file at `/var/log/supervisor/celery-stderr---supervisor.log`. Good catch.